src/app/modules/admin-feedback/__tests__/admin_feedback.service.spec.ts

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import dbHandler from '__tests__/unit/backend/helpers/jest-db'
+import { ObjectId } from 'bson-ext'
+import mongoose from 'mongoose'
+
+import getAdminFeedbackModel from 'src/app/models/admin_feedback.server.model'
+
+import { DatabaseError } from '../../core/core.errors'
+import { MissingAdminFeedbackError } from '../admin-feedback.errors'
+import * as AdminFeedbackService from '../admin-feedback.service'
+
+const AdminFeedbackModel = getAdminFeedbackModel(mongoose)
+
+describe('feedback.service', () => {
+  beforeAll(() => dbHandler.connect())
+  afterAll(() => dbHandler.closeDatabase())
+
+  describe('insertAdminFeedback', () => {
+    beforeEach(async () => {
+      await dbHandler.clearCollection(AdminFeedbackModel.collection.name)
+    })
+    afterEach(() => jest.clearAllMocks())
+
+    const MOCK_USER_ID = new ObjectId().toHexString()
+    const MOCK_RATING = 1
+    const MOCK_COMMENT = 'mock comment'
+    const MOCK_ADMIN_FEEDBACK = new AdminFeedbackModel({
+      userId: MOCK_USER_ID,
+      rating: MOCK_RATING,
+      comment: MOCK_COMMENT,
```

Comment:
just for my understanding, should there also be created and lastModified?

Reply:
created and lastModified are auto-created by mongoose!

Reply:
got it.